x-pack/test/plugin_api_integration/test_suites/task_manager/task_management.ts

```diff
@@ -794,44 +688,14 @@ export default function ({ getService }: FtrProviderContext) {
 
       await ensureTasksIndexRefreshed();
 
-      // second runNow should be successful
-      const successfulRunNowResult = runTaskNow({
+      // second runSoon should be successful
+      const successfulRunSoonResult = runTaskSoon({
         id: longRunningTask.id,
       });
 
       await provideParamsToTasksWaitingForParams(longRunningTask.id);
 
-      expect(await successfulRunNowResult).to.eql({ id: longRunningTask.id });
-    });
-
-    it('should allow a failed task to be rerun using runNow', async () => {
```

Comment:
Why aren't we allowing failed tasks to be rerun using `runSoon`?

Reply:
runSoon runs only the idle tasks. That's why i removed it.